docs/services/files.rst

```diff
@@ -13,6 +13,9 @@ All cloud files commands are based on this syntax::
 
     rack files <resource> <action> [command flags]
```

Comment:
Same thing as above about `resource`. I think I put this one in before we switched to `subservice`.